velox/functions/prestosql/Bitwise.h

```diff
@@ -28,6 +30,29 @@ struct BitwiseAndFunction {
   }
 };
 
+template <typename T>
+struct BitCountFunction {
+  // template <typename TInput>
+  FOLLY_ALWAYS_INLINE bool call(int64_t& result, int64_t num, int32_t bits) {
+    VELOX_USER_CHECK(
+        (bits > 1 && bits <= kMaxBits),
```

Comment:
nit: it would be cleaner to use symmetrical operators, e.g. bits >= 2 and bits <= kMaxBits